### PR TITLE
Dependency bump and audit fix

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Dependency bump and audit fix

#### Ticket Link
None

### CODE REVIEW

**Comment by mickmister** [comment]
Very weird lint error happening.

```
/home/circleci/project/webapp/src/components/icon.tsx
  4:8  error  'React' was used before it was defined  no-use-before-define
```

The line it's referring to is the import line, which doesn't make any sense. @jfrerich Have you run into this before?

**Comment by larkox** [comment]
After looking about it, may be a problem in one of the latest versions of typescript-eslint. If noone has a better guess, I will revert that bump.

**Comment by mickmister** [comment]
@larkox Sounds good to me

**Comment by jfrerich** [comment]
@mickmister, I've not seen that error before.  @larkox, I'm approving, with the notion that you will fix the lining error.  I'm also fine with reverting that bump.

**Review by jfrerich — APPROVED** [review]

File: webapp/src/index.tsx [hunk 1]
```diff
@@ -36,6 +30,12 @@ class Plugin {
     }
 }
 
+declare global {
+    interface Window {
+        registerPlugin(id: string, plugin: Plugin): void
+    }
+}
+
```
Comment (hanzei) [inline webapp/src/index.tsx:38]: I'm going to steal that one for https://github.com/mattermost/mattermost-plugin-starter-template :wink:

**Review by hanzei — APPROVED** [review]

File: webapp/src/index.tsx (hunk 1 above)
Comment (larkox) [reply webapp/src/index.tsx:38]: "Imitation is the sincerest form of flattery" :smile:

**Pull request merged**